content/docs/concepts/security/index.md

```diff
@@ -10,197 +10,230 @@ aliases:
     - /docs/concepts/security/rbac/
 ---
 
-Istio aims to enhance the security of microservices and their communication
-without requiring service code changes. It is responsible for:
+Istio offers strong security to the microservices and their communication.
+For example, traffic encryption to defend against the man-in-the-middle attack,
+mutual TLS and fine-grained access policies to provide flexible service access control
+and auditing to detect who did what at what time.
 
-- Providing each service with a strong identity that represents its role to
-  enable interoperability across clusters and clouds
+This page gives an overview of how Istio security can be leveraged to secure your services, wherever you run them.
+In particular, Istio security mitigates both insider and external threats against your data, endpoints, communication and platform.
 
-- Securing service to service communication and end-user to service
-  communication
+{{< image width="80%" ratio="56.25%"
+    link="./overview.svg"
+    alt="Istio security overview."
+    caption="Istio Security Architecture"
+    >}}
+
+Istio security provides strong identity, powerful policy, transparent TLS encryption and AAA (authentication/authorization/audit)
+to protect your services and data. The goals of Istio security are:
```

Comment:
The use of Istio security is very inconsistent. Do you refer to Istio Citadel? the Istio security features? both?